sequelize.ts

```diff
@@ -0,0 +1,97 @@
+import { any, date, number } from "joi";
+
+const {Sequelize, DataTypes, QueryTypes} = require('sequelize');
+const instances = require('hapi-sequelizejs').instances;
+
+// TO DO (during models): 
+// -Convert IDs to GUID/UUID once models are being used. 
+// -Autogen createdAt
+
+
+export async function GetAll(filter:string, sort: string, order: string = 'asc') {
+    // Start with a GetAll. Build as we go along by checking parameters 
+    var queryString = "SELECT * FROM Tasks ";
+
+    if (filter != null && filter != undefined)
+    {
+        queryString += `WHERE completed = ${filter} `;
+    }
+
+    if (sort != null && sort != undefined)
+    {
+        queryString += `ORDER BY ${sort} ${order}`;
+    }
+    
+    var [results] = await instances.dbs.ToDoAPI.sequelize.query(
+        queryString
+    )
+
+    return results;
+}
+
+export async function FetchTest(fetchID:number) {
+    const [results] = await instances.dbs.ToDoAPI.sequelize.query(
+        'SELECT * FROM Tasks WHERE id = ' + fetchID
+    )
+    return results;
+}
+
+export async function InsertTest(newTask:any, created:string) {         // Sequelize Autogens ID
+    const [results] = await instances.dbs.ToDoAPI.sequelize.query(
+        'INSERT INTO Tasks (taskDescription, createdAt, dueDate, completed) '+
+        `VALUES ("${newTask.taskDescription}", '${created}', '${newTask.dueDate}
+        ', ${newTask.completed})`
+    )
+    return results;
+}
+
+
+export async function Update(id:number, desc:string, due:Date, completed:boolean) {
+    const [results] = await instances.dbs.ToDoAPI.sequelize.query(
+        'UPDATE Tasks SET ' +
+        `taskDescription = "${desc}", ` +
+        `dueDate = '${due}', ` +
+        `completed = ${completed} ` +
+        `WHERE id = ${id}`
+    )
+    return results
+}
+
+export async function Delete(id:number) {
+    const [results] = await instances.dbs.ToDoAPI.sequelize.query(
+        'DELETE FROM Tasks WHERE id = ' + id
+    )
+    return results;
+}
```

Comment:
Pertaining to the discussion I've had with others to this point, it is incredibly dangerous to allow users to direct inject input into SQL queries. Doing so improperly makes your code vulnerable to an attack called [SQL injection](https://www.w3schools.com/sql/sql_injection.asp).

Making your application resistant to SQL injection is outside of the scope of your training but you should still be aware that this is very, very bad. If you must allow user input in a SQL query it is important to first [sanitize](https://www.quora.com/What-exactly-is-data-sanitization-with-respect-to-SQL-injection) by auto-escaping any special characters (most notably but not only `'` and `''`